website/content/docs/secrets/databases/index.mdx

```diff
@@ -51,7 +50,7 @@ access the associated user account in the database.
    valid.
 
    If you need to rotate root credentials, use the
-   [Rotate root credentials](vault/api-docs/secret/database/index.mdx#rotate-root-credentials)
+   [Rotate root credentials](vault/api-docs/secret/databases#rotate-root-credentials)
```

Comment:
I think we might have to use the full path here, it's giving page not found when I try to open it